### PR TITLE
Feature/dashboard event view

### PR DESCRIPTION
**Pull request by DrewBradley**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
Adds an 'event list' component to the Dashboard.
Adds a link to the header logo that redirects to the dashboard.

## *Instructions for reviewer:*
*Where should the reviewer start?*
Lines 12-60 of `src/components/dashboard/Dashboard.js`
Lines 4 and 9 of `src/components/header/Header.js`
*How should this be manually tested?*
Pull down the branch and run `npm install`.
Open the project by running `npm start`
Navigate to `localhost:3000` and see the hard coded events displayed.
*Any background context you want to provide?*
*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
Screen Shot 2021-04-13 at 5 12 32 PM
Screen Shot 2021-04-14 at 8 05 34 AM

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/components/dashboard/Dashboard.js [hunk 1]
```diff
@@ -1,10 +1,62 @@
...
+import listPlugin from '@fullcalendar/list'
+import {ListWrapper} from './ListWrapper'
+
 
 const Dashboard = () => {
   return (
     <div className="Dashboard">
       <h1>THIS IS THE Dashboard</h1>
+      <ListWrapper>
+        <FullCalendar
```
Comment (kevinhartmann23) [inline src/components/dashboard/Dashboard.js:13]: Awesomeness! Glad you were able to figure the upcoming events view. Looks great, and it's perfect for UX on dashboard!

**Review by kevinhartmann23 — APPROVED** [review]
Nice work Drew! Appreciate you!

File: src/components/dashboard/Dashboard.js [hunk 2]
```diff
@@ -1,10 +1,62 @@
...
+              plugins={[listPlugin]}
+              headerToolbar={{
+                left: '',
+                center: 'title',
+                right: ''
+              }}
+              views={{
+                upcoming: {
+                  type: 'list',
+                  duration: { days: 7 },
```
Comment (kristenmb) [inline src/components/dashboard/Dashboard.js:23]: This is great! I think seeing the upcoming week from *today* is going to be super helpful from a user perspective.

File: src/components/header/Header.js [hunk 3]
```diff
@@ -1,11 +1,14 @@
 import React from 'react'
 import logo from '../../assets/CuratourLogo.png'
 import './Header.css';
+import { Link } from 'react-router-dom'
 
 const Header = () => {
   return (
     <div className="Header">
+      <Link to="/">
```
Comment (kristenmb) [inline src/components/header/Header.js:9]: Thanks for adding this in! Love this UX

**Review by kristenmb — APPROVED** [review]

**Review by richardltyler — APPROVED** [review]
Nothing to say that hasn't been said. Looks awesome. Great job!

**Pull request merged**